lib/rubocop/cop/layout/access_modifier_indentation.rb

```diff
@@ -66,6 +66,7 @@ def autocorrect(node)
 
         def check_body(body, node)
           return if body.nil? # Empty class etc.
+          return unless body.begin_type?
```

Comment:
I think the preceding `if` is pointless now.

Reply:
removing the `return if body.nil?` makes this test fail: 

https://github.com/rubocop-hq/rubocop/blob/380c45ad36bf470ebed8d367f4cd7d3c73da8d92/spec/rubocop/cop/layout/access_modifier_indentation_spec.rb#L165-L170

I rebased again, will you be able to merge on green?

Reply:
Ah, yeah. Silly me.